atm.js

```diff
@@ -1,21 +1,63 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-}
-
-function checkingWithdrawal(amount){
-
-}
-
-function savingsDeposit(amount){
-
-}
-
-function savingsWithdrawal(amount){
-
-}
-
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+$(document).ready(function(){
+
+  var checkingDepositBtn = $("#checkingDepositBtn");
+  var checkingWithdrawalBtn = $("#checkingWithdrawalBtn");
+  var checkingDisplay = $("#checkingAccount > .balance");
+  var checkingBalance = parseInt(checkingDisplay.html().replace("$" , ""));
+  var savingsDepositBtn = $("#savingsDepositBtn");
+  var savingsWithdrawalBtn = $("#savingsWithdrawalBtn");
+  var savingsDisplay = $("#savingsAccount > .balance");
+  var savingsBalance = parseInt(savingsDisplay.html().replace("$" , ""));
+
```

Comment:
I think the key to understanding why an OOJS approach is helpful here is visualizing what the input of the constructor function would be. You've probably already guessed that in an ideal world you wouldn't have to define separate variables and methods for your checking and savings account. Wouldn't it be swell if a constructor took care of that for us, regardless of whether we're working with a checking, savings or any other account?

The below is some pseudo-code that might help you get started. Happy to take about this more during a 1:1!

```js
var Account = function( accountElement ){
  this.account = accountElement;
  this.depositButton = this.account.find(".deposit");
  // Imagine using something like the above line to select all relevant DOM elements for an account.
}
```